cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -93,10 +93,11 @@ export class PassportAuth implements Auth {
    *
    * @param defaultRedirect - location to redirect after successful authentication
    *                          when login page was loaded directly (without redirect)
+   * @param loginErrorRoute - location to redirect after unsuccessful authentication
    */
-  public authenticate(defaultRedirect: string) {
+  public authenticate(defaultRedirect: string, loginError: string) {
```

Comment:
loginErrorRoute:string?

We may add it as optional and default to login in case you do not want to redirect to any other page.
There is way for passport to redirect with some parameteres to same page.